Packaging.props

```diff
@@ -16,8 +16,18 @@
     <PackagePlatform Condition="'$(PackagePlatform)' == ''">$(Platform)</PackagePlatform>
```

Comment:
orthogonal but we should also use CoreFxExpectedPrerelease for the Microsoft.NETCore.Platforms version above in this file.